server/index.js

```diff
@@ -3,6 +3,7 @@ const app = express();
 const morgan = require('morgan');
 const parser = require('body-parser');
 const db = require('../database/index.js');
+
 let port = 3031;
```

Comment:
Do you change your port anywhere? Is it best practice to use a `let` for our ports? This seems like it could be a `const` to me.